src/main/java/com/codeborne/selenide/webdriver/FirefoxDriverFactory.java

```diff
@@ -23,6 +23,7 @@ boolean supports() {
 
   @Override
   WebDriver create(final Proxy proxy) {
+    System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");
```

Comment:
System.getProperty("selenide.pollingInterval", "100"))

Reply:
We need to decide - do we really want to provide way to configure this property.
If yes - we have to introduce new configuration field.
If not - just hide this with default value.

Because if use suggested code - it will be undocumented configuration.

Reply:
I vote for not adding new configuration field. Let's just use system property. Yes, it will be an "undocumented" feature, but almost nobody does really need to change it.